src/sql_builder/mod.rs

```diff
@@ -129,22 +129,76 @@ impl Bind for &str {
     }
 }
 
-#[test]
-fn it_builds_sql() {
-    #[allow(dead_code)]
+impl<'a, T: Bind> Sealed for &'a [T] {}
+
+const EMPTY_LIST_SQL: &str = "(NULL)";
```

Comment:
Haven't we decided to use arrays instead, as you had suggested?

Reply:
Done